Source/WebCore/animation/AnimationTimeline.h

```diff
@@ -36,6 +37,17 @@ namespace WebCore {
 class AnimationTimelinesController;
 class WebAnimation;
 
+enum class NamedTimelineRange { Normal, Omitted, Cover, Contain, Entry, Exit, EntryCrossing, ExitCrossing };
```

Comment:
We could call this `TimelineRangeName` to match the spec type `<timeline-range-name>`.

Reply:
Fixed.